app/services/forest_liana/filters_parser.rb

```diff
@@ -81,11 +81,14 @@ def parse_condition(condition)
       parsed_field = parse_field_name(field)
       parsed_operator = parse_operator(operator)
       parsed_value = parse_value(operator, value)
+      field_and_operator = "#{parsed_field} #{parsed_operator}"
 
-      if Rails::VERSION::MAJOR >= 5
-        ActiveRecord::Base.sanitize_sql(["#{parsed_field} #{parsed_operator} ?", parsed_value])
+      if Rails::VERSION::MAJOR < 5
+        "#{field_and_operator} #{ActiveRecord::Base.sanitize(parsed_value)}"
+      elsif Rails::VERSION::MAJOR == 5 && Rails::VERSION::MINOR == 1
```

Comment:
[MISC] A small comment here to state the reason of this specific case could be cool. As I understood:
- `sanitize` removed for Rails 5.1
- `sanitize_sql` introduced in 5.2